frontend/packages/console-shared/src/components/formik-fields/TextColumnField.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import { FieldArray, useField } from 'formik';
+import { Flex, FlexItem, FlexModifiers, FormGroup, TextInputTypes } from '@patternfly/react-core';
+import { MinusCircleIcon } from '@patternfly/react-icons';
+import { global_disabled_color_200 as disabledColor } from '@patternfly/react-tokens';
+import { InputField, useFormikValidationFix } from '@console/shared';
+import MultiColumnFieldFooter from './multi-column-field/MultiColumnFieldFooter';
+import { getFieldId } from './field-utils';
+import { FieldProps } from './field-types';
+
+export interface TextColumnFieldProps extends FieldProps {
+  required?: boolean;
+  isValid?: boolean;
```

Comment:
Why are you taking `isValid` prop from parent? You should make use of errors and touched object from `useField` like all other fields.